v3/src/components/data-display/hooks/use-data-tips.ts

```diff
@@ -73,11 +74,12 @@ export const useDataTips = ({dotsRef, dataset, displayModel}: IUseDataTips) => {
 
     // support disabling data tips via url parameter for jest tests
     if (urlParams.noDataTips === undefined) {
-      dotsRef.current && select(dotsRef.current)
-        .on('mouseover', showDataTip)
-        .on('mouseout', hideDataTip)
-        .call(dataTip)
+      // TODO PIXI: reimplement data tips
```

Comment:
Comment out or add `eslint-disable` comments to eliminate eslint warnings for unused `showDataTip` and `hideDataTip` functions above.